tricoro.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
     <title>IIDX tricoro DLL Modder</title>
-    <link rel="stylesheet" href="css/style.css">
+    <script src="/cdn-cgi/apps/head/HwU4Afx3ZsFv3HCMpwtQftaMMBI.js"></script><link rel="stylesheet" href="css/style.css">
```

Comment:
You've accidentally copied in the "inserted by cloudflare" script

Reply:
fixed

Reply:
You left extra whitespace but eh whatever